thoth/common/workflows.py

```diff
@@ -317,3 +317,28 @@ def submit_inspection_workflow(
         )
 
         return workflow_id
+
+    def submit_adviser_workflow(
+        self,
+        adviser_id: str,
+        template_parameters: Optional[Dict[str, str]] = None,
+        workflow_parameters: Optional[Dict[str, Any]] = None,
+    ) -> str:
+        """Submit Adviser Workflow."""
+        if not self.openshift.amun_infra_namespace:
+            raise ConfigurationError("Infra namespace was not provided.")
+
+        template_parameters = template_parameters or {}
+        workflow_parameters = workflow_parameters or {}
+
+        template_parameters["THOTH_ADVISER_JOB_ID"] = adviser_id
+
+        workflow_id: str = self.submit_workflow_from_template(
+            self.openshift.amun_infra_namespace,
+            label_selector="template=adviser",
+            template_parameters=template_parameters,
+            workflow_parameters=workflow_parameters,
+            workflow_namespace=self.openshift.amun_infra_namespace
```

Comment:
Should the adviser workflow be run in the Amun Infra namespace?

Reply:
I dont think so
@fridex

Reply:
Should be `thoth-backend-stage` iirc.

Reply:
`thoth-backend-stage`

Reply:
Thanks:) I will correct it, I was still testing in test env.